content/blog/eng/php-day-2020.md

```diff
@@ -0,0 +1,78 @@
+
+I would like to thanks the folks at [Grusp](https://www.grusp.org/) for their passionate work. Try to sustain them!
+
+What follows is a mere recap of the event, with my little bit of comments. 
+
+
+# Things I Wish I Knew When I Started Playing Developer
+## by Gary Hockin ([@GeeH](https://twitter.com/GeeH))
+
+
+A trip into developers' mindset from a ludic (and hence, engaging) viewpoint. Gary is a natural born speaker ;) and this talk deserves to be seen by anyone interested in programming, no matter their background and interests, at least once. 
+
+#What's new in PHP 8 
+## Derick Rethans ([@derickr](https://twitter.com/derickr))
+
+Personally, I did not like the style of the presentation (not the theme of slides, but how they got presented! :D) but the information provided is very important and so you should take a look here: https://derickrethans.nl/talks/php-phpday20 
+
+#Symfony 5, the new bit 
+## Andreas Hucks ([@meandmymonkey](https://twitter.com/meandmymonkey))
+
+A deep dive into Symfony 5 features and enhancements. Knowing where the framework is heading will make us take better-informed decisions in our day to day work.
+
+#Leveraging Typed Exceptions for Cleaner Error Handling 
+## Chris Holland ([@chrisholland](https://twitter.com/chrisholland))
+
+A talk suited mostly to entry-level/mid-level developers, but indeed useful to learn (or recap) how to establish a lifelong relationship with exceptions: they are not enemies, they can help us contextualizing the scope in which they occur and so speed up the debugging process.
+Slides available: https://bit.ly/exceptions-ftw
+ 
+#Kubernetes for PHP developers 
+## Alessandro Lai ([@AlessandroLai](https://twitter.com/AlessandroLai))
+
+A well-presented talk by my colleague Alessandro - Too much dashboard screenshot for me, but that's only because I am more dev than ops ;)
+
+#Technically Speaking: Improve your code with documentation 
+## Alexandra White ([@heyawhite](https://twitter.com/heyawhite))
+
+Brilliantly presented by Alexandra! A tidbit that I brought home: there are moments to be (like) Shakespeare and others not. One of those when not to be Shakespeare is when writing documentation.
+I must say that this talk lighted my interest in technical writing.
+The cherry on the cake was that the presentation had subtitles, which should definitely become a de-facto standard at least for pre-recorded videos. Very inclusive!
+Slides: https://docs.google.com/presentation/d/1l7cjrouAGdsGae4nXyJLcGcool1l0zx5knXvL_4h04Q/edit#slide=id.g589a77319a_0_34
+
+#Looping the Loop with SPL Iterators 
+## Mark Baker ([@Mark_Baker](https://twitter.com/Mark_Baker))
+
+A tour of the not-well-known iterators world and a little mea-culpa about the initially almost-not-existent SPL documentation! 🙂 
```

Comment:
What do you mean by "initially"?

Reply:
A tour of the not-well-known iterators world. There was also time for a little mea-culpa about the almost-not-existent documentation when the library was first published and its initial years.